src/iccpd/src/iccp_ifm.c

```diff
@@ -335,51 +423,133 @@ static void do_ndisc_learn_from_kernel(struct ndmsg *ndm, struct rtattr *tb[], i
                     ndisc_msg->mac_addr[0], ndisc_msg->mac_addr[1], ndisc_msg->mac_addr[2], ndisc_msg->mac_addr[3], ndisc_msg->mac_addr[4],
                     ndisc_msg->mac_addr[5]);
 
+
+    if (msgtype != RTM_DELNEIGH) {
+        if (memcmp(ndisc_lif->ipv6_addr, addr_null, 16) == 0) {
+            ICCPD_LOG_DEBUG(__FUNCTION__, "IPv6 address not configured on %s, ignore ND", ndisc_lif->name);
+            return;
+        }
+    }
+
+    uint8_t bcast_mac[ETHER_ADDR_LEN] = {0xff, 0xff, 0xff, 0xff, 0xff, 0xff};
+    if (memcmp(ndisc_msg->mac_addr, bcast_mac, ETHER_ADDR_LEN) == 0)
+    {
+        ICCPD_LOG_DEBUG(__FUNCTION__, "Ignoring neighbor entry due to bcast lladdr");
+        msgtype = RTM_DELNEIGH;
```

Comment:
should you return here or continue for this case?

Reply:
@gechiang : Thanks , returned while ignoring the event.!